lib/fizzy/execute.cpp

```diff
@@ -565,6 +566,37 @@ inline T ftrunc(T value) noexcept
     return std::trunc(value);
 }
 
+template <typename T>
+T fnearest(T a) noexcept
+{
+    // This implementation uses std::nearbyint() which rounds to integer using current rounding
```

Comment:
Add static assert that T is floating point?

Reply:
Added.